tests/generate_fixtures/generate_pyani_anim_matrices.sh

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
+
+# The MIT License
+#
+# Copyright (c) 2024 University of Strathclyde
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+
+#create db
+pyani createdb --force
+
+#index with pyANI v0.3
+pyani index -i ../../tests/fixtures/viral_example/
```

Comment:
I would suggest making symlinks to the FASTA files in the temp working folder, that way there are no stray ``*.md5`` files etc to clean up.

On linux this would be just ``cp -s ../../tests/fixtures/viral_example/*.f* .`` but on macOS you'd have to use ``ln`` in a loop:

```bash
for FASTA in ../../fixtures/viral_example/*.f*; do ln -s $FASTA ${FASTA##*/} ; done
```

(This would need adjustment if you want to put the FASTA links in a subfolder - ask)

I assume pyANI is also fine with ``.fasta``, ``.fas``, and ``.fna`` here?

Reply:
The `ln` loop worked fine :)

Yes, `pyANI v0.3` works on these extensions so there is no problem with that at all.